ED/protocol.html

```diff
@@ -1466,8 +1462,8 @@ <h2 property="schema:name">Changelog</h2>
                   </tr>
                   <tr about="#d18392b9-8ffc-4264-a5a2-10e2da6fac32" typeof="spec:Change">
                     <td><a href="https://www.w3.org/2023/Process-20230612/#class-4" rel="spec:changeClass" resource="spec:ChangeClass4">4</a></td>
-                    <td><a href="#server-content-type-payload" rel="spec:changeSubject">#server-content-type-payload</a></td>
-                    <td property="spec:statement">Add requirement for server to include <code>Content-Type</code> in <a href="#server-content-type-payload">messages with payload</a>.</td>
+                    <td><a href="#server-content-type" rel="spec:changeSubject">#server-content-type</a></td>
+                    <td property="spec:statement">Add requirement for server to include <code>Content-Type</code> in <a href="#server-content-type">messages with content</a>.</td>
```

Comment:
See https://github.com/solid/specification/pull/561/files#r1306557501

```suggestion
                    <td><a href="#server-content-type-content" rel="spec:changeSubject">#server-content-type</a></td>
                    <td property="spec:statement">Add requirement for server to include <code>Content-Type</code> in <a href="#server-content-type-content">messages with content</a>.</td>
```

Reply:
If more specificity is required, let us go with something more informative than repeating a word. For example: `#server-content-type-required`.